api/requirements.txt

```diff
@@ -8,6 +8,7 @@ psycopg2==2.9.5
 django-trench==0.3.1
 django-phonenumber-field[phonenumbers]
 secure==0.3.0
+Pillow==9.4.0
```

Comment:
Do we really need this ? I don't see you calling this library anywhere (I might be blind)

Reply:
Yes it's important for ImageField.